### PR TITLE
[MAZ-108] [MAZ-235] Implement Add / Update Email, and Username, API Integration for Email, Username, and Eth-address settings pages 

### PR DESCRIPTION
**Pull request by kriszzy01**

### CODE REVIEW

File: pages/settings/account/email.tsx [hunk 1]
```diff
@@ -1,9 +1,140 @@
...
+// import { useSWRConfig } from 'swr';
+import { useAccount, useSignMessage } from 'wagmi';
+import { Button, Input, Modal, SettingsLayout, Spinner } from 'components';
+// import { useProfile } from 'hooks';
+import { getMessageToBeSigned, getProfile, updateProfile } from 'utils/api';
+
+type Steps = 'idle' | 'active' | 'error';
 
 const EmailPage: NextPage = () => {
+  // const { mutate } = useSWRConfig();
```
Comment (Dhaiwat10) [inline pages/settings/account/email.tsx:13]: Is this something that you still need to implement? If not, let's remove all the commented-out code that we don't need!

Comment (Dhaiwat10) [reply pages/settings/account/email.tsx:13]: (also the imports)

File: pages/settings/account/eth-address.tsx [hunk 2]
```diff
@@ -47,13 +58,13 @@ const EthAddressPage: NextPage = () => {
...
+                    </div> */}
                   </div>
                 ))}
               </div>
 
-              <Button className="w-full uppercase" size="large">
-                CONNECT WALLET
+              <Button className="w-full uppercase" size="large" disabled={true}>
+                CONNECT WALLET (COMING SOON)
               </Button>
```
Comment (Dhaiwat10) [inline pages/settings/account/eth-address.tsx:68]: Have we decided to not implement this for now?

**Review by Dhaiwat10 — COMMENTED** [review]
Looks good! Just added a couple of questions for you. Let's get these out of the way and we can merge :D

File: pages/settings/account/eth-address.tsx (hunk 2 above)
Comment (kriszzy01) [reply pages/settings/account/eth-address.tsx:68]: This was instructions from Sebastian and Woj, that's why it was done this way.

**Pull request merged**